docs/Users_Guide/release-notes.rst

```diff
@@ -76,6 +76,351 @@ METplus Version 6.0.0 Beta 1 Release Notes (2023-09-15)
+METplus v4.0.0 introduced a more efficient approach to overriding values in
+MET configuration files through the METplus wrappers.
+See :ref:`metplus-control-met` for more information.
+
+Prior to the v4.0.0 release, overriding MET settings that were not yet
+supported by METplus configuration variables required users to copy an
+existing *wrapped* MET config file, make the desired modifications,
+then update their METplus config file to use the user-defined MET
+configuration file.
+
+The new approach removes the need to maintain multiple *wrapped* MET
+configuration files by using the *wrapped* MET configuration files that
+are provided with the METplus wrappers.
+This allows any new METplus configuration variables that set MET variables
+to automatically be supported when moving to a new version of METplus.
+Any MET configuration settings that are not yet controlled by a corresponding
+METplus configuration variable can easily be set in a METplus configuration
+file by using the MET config overrides variables.
+See :ref:`met-config-overrides` for more information.
+
+How to tell if upgrade is needed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Prior to v6.0.0, a use case that uses a wrapped MET config file that is
+out-of-date from the version provided with the METplus wrappers will report a
+warning in the log output alerting the user that an expected environment
+variable is not found::
+
+    WARNING: Environment variable ${METPLUS_MODEL} is not utilized in MET config file: /path/to/GridStatConfig_trey
+
+This is often an indicator that the use case will need to be updated.
+The deprecated environment variables, e.g. **${MODEL}**, were still set by the
+wrappers, so the use case still ran without any issues.
+
+Starting in v6.0.0, the deprecated environment variables are no longer set and
+an error message will be displayed for each deprecated variable that was found::
+
+    ERROR: Deprecated environment variables found in MET config file: /path/to/GridStatConfig_trey
+    ERROR: Deprecated environment variable ${MODEL} found
+    ERROR: Deprecated environment variable ${OBTYPE} found
+    ERROR: Deprecated environment variable ${REGRID_TO_GRID} found
+
+If these errors occur,
+the use case will not run until the METplus configuration file has been updated.
+
+How to upgrade
+^^^^^^^^^^^^^^
+
+Removing **<WRAPPER_NAME>_CONFIG_FILE**, e.g. :term:`GRID_STAT_CONFIG_FILE`,
+from the METplus config file will prevent the errors and
+allow the use case to run.
+However, this alone may result in changes to the output because the settings
+in the user-defined wrapped MET config file may no longer be set.
+
+**It is important to carefully review the settings and set the appropriate
+METplus configuration variables to preserve the original configuration!**
+
+Compare the user-defined wrapped MET config file (:term:`GRID_STAT_CONFIG_FILE`)
+with the default config file that is found in the MET installation location,
+e.g. share/met/GridStatConfig_default.
```

Comment:
@georgemccabe What do you think about adding MET_INSTALL_DIR to the beginning of share/met/GridStatConfig_default?  I wasn't sure if the formatting should be ${MET_INSTALL_DIR}/share/met/GridStatConfig_default or {ENV[MET_INSTALL_DIR]}/share/met/GridStatConfig_default, which is why I did not directly suggest a change.

Reply:
There is no environment variable called MET_INSTALL_DIR that would be set for the user, but there is a METplus config variable with that name, so I think it may be misleading to reference it here. I do see what you are saying about making it more clear how to find the path. I suggest we either change it to:

`<MET_INSTALLATION_LOCATION>/share/met/GridStatConfig`

or

`met-X.Y.Z/share/met/GridStatConfig`

or

`/path/to/met-X.Y.Z/share/met/GridStatConfig`

What do you think?

Reply:
I prefer `/path/to/met-X.Y.Z/share/met/GridStatConfig`.